editors_checklist.md

```diff
@@ -0,0 +1,51 @@
+# Editors' checklist
+
+Upon submission:
+
+-   Check that template has been properly filled out
+-   Tag issue appropriately with `package`, `topic:`, etc. tags
+-   Check against policies for [fit](policies.md#fit) and [overlap](policies.md#fit)
+    If reject, close issue, direct authors to other groups/repos as appropriate.
+-   Check that mandatory parts of template are complete.  If not, close issue,
+    direct authors toward appropriate instructions.
+-   Assign reviewers.  Use the #onboarding slack channel for discussion about
+    potential reviewers.  Assign reviewer by tagging in the issue with the
+    following format to activate  the automated reminders bot:
+   
+      Reviewers: @githubname 
+      Due date: YYYY-MM-DD
+
+During review:
+
+-   Check in with reviewers and authors occaisionally.
+
+After review:
```

Comment:
Also here:  On reviews being submitted, change the review tag to `review-in-awaiting-changes`